### ISSUE TITLE
Configure Android Lint

### ISSUE THREAD

**Issue by wb9688** [TeamNewPipe/NewPipe#3526]
We should configure Android Lint in its `lint.xml` configuration to ignore stupid things like `MissingTranslation`, then fix remaining issues, and make builds fail when there's a `Fatal` or an `Error`, similar to the Checkstyle stuff.

**Comment by opusforlife2** [comment]
@wb9688 Is this different from that other linter issue?

**Comment by wb9688** [comment]
@opusforlife2: Yes, this one catches common Android-related errors, such as incorrect translations (that would cause crashes).

**Comment by ShareASmile** [comment]
Done by theimpulson.
